src/shared/footer.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+  HoverCard,
+  HoverCardContent,
+  HoverCardTrigger,
+} from '@components';
+import { useTranslations } from 'next-intl';
+import Image from 'next/image';
+import Link from 'next/link';
```

Comment:
Используй враппер от next-intl.
```suggestion
import Link from '@i18n';
```

Reply:
тут Link используется для внешних ссылок. хотелось бы оставить next/link, а вот для внутренней навигации использовать import Link from '@i18n'; (исправлю в Header)